tests/test_fingerprints.py

```diff
@@ -4,7 +4,7 @@
 
 import pytest
 import scrapy
-from scrapy.dupefilters import request_fingerprint
+from scrapy.utils.request import request_fingerprint
```

Comment:
We are still using a deprecated function, but supporting the new request fingerprinting may require more profound changes.

I wonder if maybe we can keep `request_fingerprint` around for longer than the minimum 1-year deprecation period on the Scrapy side.